### ISSUE TITLE
ideal protein concentration?

### ISSUE THREAD

**Issue by andrewkern** [andrewkern/bagels#3]
over on #2 @joelmcg brings up the suggestion of using high gluten flours like Sir Lancelot or All Trumps, and notes

> Sir Lancelot has 14.2% protein. King Arthur All-Purpose flour has 11.7%, VWG 75%.
> Using 1 c = 128 g:
> 992 g SL * 0.142 = 141 g protein
> 949 g AP * 0.117 + 43 g VWG * 0.75 = 143 g protein

I wonder how much higher we could push this? some of the bagels that I grew up with were like lead weights

**Comment by joelmcg** [comment]
I suspect many NY bagel shops use All Trumps. If you wanted a denser bagel, you might push the hydration down even further. Reinhart BBA is 57%, but I've seen recipes that go as low as 50%. At that point it might be a challenge to incorporate all the flour, but it gets you closer to lead weight territory.

**Comment by andrewkern** [comment]
i'm not sure that commercial NYC places use nice flour like that, but i should do more research. in my experience the ingredients tend to be super cheap in most "real" bagel places.

**Comment by joelmcg** [comment]
Here is Russ & Daughters using Sir Lancelot https://www.youtube.com/watch?v=g_t97SW0Yug

**Comment by jamestwebber** [comment]
I usually use King Arthur's bread flour, but some of the best bagels (and bread/pizza) I've made was with [Graincraft Power](https://www.graincraft.com/products/power/), which I bought in a 25 lb bag in early 2020 because bread flour was impossible to find.

In bulk this stuff is not that expensive--insignificant compared to the cost of labor.

**Comment by andrewkern** [comment]
> Here is Russ & Daughters using Sir Lancelot https://www.youtube.com/watch?v=g_t97SW0Yug

not surprised that R&D uses this-- they are high end, and not really a bagel joint per se.

**Comment by andrewkern** [comment]
> I usually use King Arthur's bread flour, but some of the best bagels (and bread/pizza) I've made was with [Graincraft Power](https://www.graincraft.com/products/power/), which I bought in a 25 lb bag in early 2020 because bread flour was impossible to find.
> 
> In bulk this stuff is not that expensive--insignificant compared to the cost of labor.

@jamestwebber that flour looks interesting, but even lower % gluten

**Comment by jamestwebber** [comment]
True, it isn't the very highest protein out there (I can't remember if I juiced it with additional gluten). Gluten isn't _everything_ though. 😛
